ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -270,6 +271,14 @@ def get_action(
         )
 
         self.save_memories(global_agent_ids, run_out.get("memory_out"))
+        # For Compatibility with buffer changes for hybrid action support
+        if "log_probs" in run_out:
```

Comment:
I know it's not new to this PR, but should we move these strings to constant/enums? "pre_action" and "actions_pre" has me scared, and "continuous" is generally pretty typo-prone.